torchvision/datasets/fer2013.py

```diff
@@ -47,14 +52,25 @@ def __init__(
                 f"https://www.kaggle.com/c/challenges-in-representation-learning-facial-expression-recognition-challenge"
             )
 
+        pixels_key = " pixels" if use_icml else "pixels"  # yes, for real
```

Comment:
Oh FFS. Took me two minutes to notice the extra space. I share your feelings here, but maybe point out the space in the comment?